cpp/src/io/parquet/reader_impl.cpp

```diff
@@ -440,4 +440,28 @@ bool reader::impl::has_next()
   return _current_read_chunk < _chunk_read_info.size();
 }
 
+namespace {
+static parquet_column_schema walk_schema(aggregate_reader_metadata const* mt, int idx)
```

Comment:
```suggestion
parquet_column_schema walk_schema(aggregate_reader_metadata const* mt, int idx)
```

Reply:
why not `static`?

Reply:
got it. Thanks
https://stackoverflow.com/questions/4422507/superiority-of-unnamed-namespace-over-static

Reply:
As far as I can tell, it's not a member function of a class, so there is no static internal state you are relying on.

static on free functions prevents leakage across multiple TUs, but using an anonymous namespace provides the same benefit and is more in line with other libcudf code.

What were your reasons for using static, if it's something different?